docs/tutorial/filter.md

```diff
@@ -0,0 +1,34 @@
+# Running Filter Protocol
+
+## How to
+
+Build:
+
+```
+# make wakunode2 is run as part of scripts2 target
+make scripts2
+```
+
+Run two nodes and connect them:
+
+```
+# Starts listening on 60000 with RPC server on 8545.
+# Note the "listening on address" in logs.
+./build/wakunode2 --ports-shift:0
+
+# Run another node with staticnode argument
+./build/wakunode2 --ports-shift:1 --staticnode:/ip4/0.0.0.0/tcp/60000/p2p/16Uiu2HAmF4tuht6fmna6uDqoSMgFqhUrdaVR6VQRyGr6sCpfS2jp --filternode:/ip4/0.0.0.0/tcp/60000/p2p/16Uiu2HAmF4tuht6fmna6uDqoSMgFqhUrdaVR6VQRyGr6sCpfS2jp
+```
+
+You should see your nodes connecting.
+
+Do basic RPC calls:
+
+```
+./build/rpc_subscribe 8545
```

Comment:
I don't understand how you specify that you want to be a "light node" and why we are subscribing here

Reply:
Even if we don't do this, is it 100% the node isn't subscribing by default?